tutorials/proto_rename/PersonTool.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tutorials.proto_rename;
+
+import com.google.startupos.common.FileUtils;
+import com.google.startupos.tutorials.proto_rename.Protos.Person;
+
+public class PersonTool {
+
+  private static final String PERSON_PROTOBINARY_PATH;
+
+  static {
+    FileUtils fileUtils = DaggerProtoRenameComponent.create().getFileUtils();
+    PERSON_PROTOBINARY_PATH = fileUtils.joinToAbsolutePath("person.pb");
```

Comment:
Where does it write it?
If it's not in the current directory, maybe you can use `fileUtils.getCurrentWorkingDirectory()`?

Reply:
It was written not in the current directory. Fixed it.